### ISSUE TITLE
Manual Sync task shows error

### ISSUE THREAD

**Issue by griffon56** [AzuraCast/AzuraCast#531]
- **Installation Method:  Docker
- **Host OS (if using Docker): Ubuntu 16.04

Hello,
How to get rid of the following error message that I get when I try to manually run a sync task 

`Your proxy directory "/var/azuracast/www_tmp/proxies" must be writable`

and 

```
500
Your proxy directory "/var/azuracast/www_tmp/proxies" must be writable
UnexpectedValueException.php : 40
```

BTW 3 out of 4 task are not run anymore.
![image](https://user-images.githubusercontent.com/25333002/38146111-cff24ee8-344c-11e8-8f6b-45077d58b30e.png)

**Comment by griffon56** [comment]
Latest docker-update did run nearly error-free
```

Creating network "azuracast_default" with the default driver
Creating azuracast_mariadb_1 ...
Creating azuracast_stations ...
Creating azuracast_redis_1 ...
Creating azuracast_mariadb_1
Creating azuracast_influxdb_1 ...
Creating azuracast_influxdb_1
Creating azuracast_stations
Creating azuracast_stations ... done
Creating azuracast_web_1 ...
Creating azuracast_web_1 ... done
2018/03/30 19:36:30 Waiting for: tcp://mariadb:3306
2018/03/30 19:36:30 Waiting for: tcp://influxdb:8086
2018/03/30 19:36:30 Connected to tcp://mariadb:3306
2018/03/30 19:36:30 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:31 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:32 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:33 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:34 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:35 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:36 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:37 Problem with dial: dial tcp 172.18.0.3:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:36:38 Connected to tcp://influxdb:8086
Updating AzuraCast (Environment: production)
HEAD is now at 848dc9e #523 -- Smarter handling of timezones, especially for playlists
Already up-to-date.
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Loading composer repositories with package information
Installing dependencies from lock file
Nothing to install or update
Generating autoload files
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
Router cache file cleared.
Local cache flushed.
Loading configuration from the integration code of your framework (setter).

                    Doctrine Database Migrations


No migrations to execute.

 Processing entity "Entity\StationMedia"
 Processing entity "Entity\User"
 Processing entity "Entity\RolePermission"
 Processing entity "Entity\StationPlaylist"
 Processing entity "Entity\StationWebhook"
 Processing entity "Entity\Analytics"
 Processing entity "Entity\Settings"
 Processing entity "Entity\StationMount"
 Processing entity "Entity\Listener"
 Processing entity "Entity\Role"
 Processing entity "Entity\Song"
 Processing entity "Entity\StationMediaArt"
 Processing entity "Entity\SongHistory"
 Processing entity "Entity\StationStreamer"
 Processing entity "Entity\ApiKey"
 Processing entity "Entity\Station"
 Processing entity "Entity\StationRequest"

 Proxy classes generated to "/var/azuracast/www_tmp/proxies"
Restarting all radio stations...
Restarting station #3: My radio
2018/03/30 19:36:51 Command finished successfully.
Creating azuracast_letsencrypt_1 ...
Creating azuracast_letsencrypt_1
azuracast_influxdb_1 is up-to-date
azuracast_redis_1 is up-to-date
azuracast_mariadb_1 is up-to-date
azuracast_stations is up-to-date
azuracast_web_1 is up-to-date
Creating azuracast_nginx_1 ...
Creating azuracast_cron ...
Creating azuracast_cli_1 ...
Creating azuracast_nginx_1
Creating azuracast_cron
Creating azuracast_nginx_1 ... done
```

**Comment by griffon56** [comment]
trying to update manually is another story
```
root@vps-25068:/docker# docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
root@vps-25068:/docker# docker-compose run --rm cli azuracast_update
Creating network "azuracast_default" with the default driver
Creating azuracast_mariadb_1 ...
Creating azuracast_stations ...
Creating azuracast_redis_1 ...
Creating azuracast_mariadb_1
Creating azuracast_influxdb_1 ...
Creating azuracast_redis_1
Creating azuracast_stations
Creating azuracast_stations ... done
Creating azuracast_web_1 ...
Creating azuracast_web_1 ... done
2018/03/30 19:53:01 Waiting for: tcp://mariadb:3306
2018/03/30 19:53:01 Waiting for: tcp://influxdb:8086
2018/03/30 19:53:01 Connected to tcp://mariadb:3306
2018/03/30 19:53:01 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:02 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:03 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:04 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:05 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:06 Problem with dial: dial tcp 172.18.0.4:8086: connect: connection refused. Sleeping 1s
2018/03/30 19:53:07 Connected to tcp://influxdb:8086
Updating AzuraCast (Environment: production)
HEAD is now at 848dc9e #523 -- Smarter handling of timezones, especially for playlists
error: cannot open .git/FETCH_HEAD: Permission denied

Loading composer repositories with package information
Installing dependencies from lock file
Nothing to install or update
Generating autoload files
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
Router cache file cleared.
Local cache flushed.
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityRole.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStation.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntitySettings.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationWebhook.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityApiKey.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationMediaArt.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationRequest.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityListener.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationMedia.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntitySongHistory.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationStreamer.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationPlaylist.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityAnalytics.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityUser.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityRolePermission.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntitySong.php': Permission denied
rm: cannot remove '/var/azuracast/www_tmp/proxies/__CG__EntityStationMount.php': Permission denied
Loading configuration from the integration code of your framework (setter).

                    Doctrine Database Migrations


No migrations to execute.

In GenerateProxiesCommand.php line 87:

  Proxies destination directory '<info>/var/azuracast/www_tmp/proxies</info>' does not have write permissions.


orm:generate-proxies [--filter FILTER] [--] [<dest-path>]

Restarting all radio stations...
Restarting station #3: Pacoje radio

In UnexpectedValueException.php line 40:

  Your proxy directory "/var/azuracast/www_tmp/proxies" must be writable


azuracast:radio:restart

2018/03/30 19:53:12 Command exited with error: exit status 1

```

**Comment by griffon56** [comment]
This was fixed with the help of topic #518
`docker-compose run --rm cli`

then, inside the container, delete following folder 

`/var/azuracast/www_tmp/proxies`
